README.md

```diff
@@ -17,6 +17,17 @@
 -  `container_tag`: The container tag to scan against (default: `latest`)
 -  `crowdstrike_region`: The CrowdStrike Cloud region to submit for scanning (default: `us-1`)
 
+### Optional input for vulnerability score threshold
+
+-  `crowdstrike_score`: The score threshold used to allow for step success (default: `500`)
```

Comment:
Let's add this under the existing inputs section. Could add `Optional` to in the default parenthesis like ```(Optional. default: `500`)```